### ISSUE TITLE
Feature Request: Add Notification::Type_SceneEvent

### ISSUE THREAD

**Issue by jbardi** [belyalov/open-zwave-mqtt#12]
Still having to use my current openzwave setup because I didn't realize your program does not yet support Zwave Scene Events, and as I have 10 Aeotec MiniMotes around the house to control different device scenes, I would need the SceneEvent notification to trigger the "scene_activation" command class (0x2B)

I know you probably wrote this for your own personal purposes and shared it for others to use, so you most likely do not have any Zwave MiniMote controllers, but if you find the time in future, I'd like to put in a feature request for support of the SceneEvent notification and scene_activation command class. I've been messing with the source code trying to crudely implement it myself, but although I am a programmer, C++ is not my strong suit, so it's not going as smoothly as I thought it might lol

Anyway, thanks again for the program!

**Comment by belyalov** [comment]
Yeah, you're right. This FR sounds totally reasonable to me, I think it is easy to implement, but I don't have such devices yet...

Anyway I can try to add support for scenes if you help me to test it?

**Comment by jbardi** [comment]
Of course, I would gladly help to test it out! Scene events are the only thing keeping me from moving away from the Node-Red Zwave nodes I am currently using, and as I make a lot of changes in Node-Red, every single time I restart Node-Red, the Zwave library has to reinitialize which takes sometimes up to an hour for my motion sensors to fully be engaged, so I am willing to do anything I can do to get my Zwave service out of Node-Red so it can run independently.

**Comment by belyalov** [comment]
@jbardi is it still needed for you? Looks like I'll have free time soon and may look into adding support for scenes.
Can you recommend device or describe usual use cases to start with?
